packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -124,16 +114,9 @@ export class DocumentService
 		}
 
 		const getStorageManager = async (disableCache?: boolean): Promise<GitManager> => {
-			const shouldUpdateDiscoveredSessionInfo = this.shouldUpdateDiscoveredSessionInfo();
-			if (shouldUpdateDiscoveredSessionInfo) {
-				await this.refreshDiscovery();
-			}
-			if (
-				!this.storageManager ||
-				!this.noCacheStorageManager ||
-				shouldUpdateDiscoveredSessionInfo
-			) {
-				if (shouldUpdateDiscoveredSessionInfo) {
+			const refreshed = await this.maybeRefreshDiscovery();
```

Comment:
>refreshed [](http://example.com/codeflow?start=9&length=9)

`isRefreshed` more clearly depicts it's a boolean

Reply:
`isRefreshed` sounds a bit wonky to me. I'll stick with `refreshed`